modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -191,6 +196,16 @@ definition {
 		return ${response};
 	}
 
+	macro getObjectEntryById {
```

Comment:
Thanks for the getObjectEntryById {} macro, we have an refactor task that moving all customObjectEntries from ObjectDefinitionAPI to CustomObjectAPI. Could you try to use the existing ObjectDefinitionAPI. _getObjectEntryById() or ObjectDefinitionAPI.getObjectEntries() so that we won't accumulate more to the refactor task? Does this sounds ok?

Reply:
A method 'createObjectEntryWithName' in ObjectDefinitionAPI.macro uses this ObjectDefinitionAPI. _getObjectEntryById and too many cases have 'createObjectEntryWithName', maybe this time I can move this method 'ObjectDefinitionAPI. _getObjectEntryById' to CustomObjectAPI

Reply:
Sounds good, then we have less tasks when doing the refactor task, thanks :smile: